sdk/examples/client/block/01_block_confirmation_time.rs

```diff
@@ -60,19 +60,5 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         block_id
     );
 
-    // TODO uncomment when we have a new confirmation logic
-    // Get the block metadata.
-    // let metadata = client.get_block_metadata(&block_id).await?;
-
-    // if let Some(ms_index) = metadata.referenced_by_milestone_index {
-    //     let ms = client.get_milestone_by_index(ms_index).await?;
-    //     println!(
-    //         "Block {block_id} got confirmed by milestone {ms_index} at timestamp {}.",
-    //         ms.essence().timestamp()
-    //     );
-    // } else {
-    //     println!("Block {block_id} is not confirmed.")
-    // }
-
```

Comment:
Not sure there is actually an equivalent to this tbh

Reply:
Maybe at L53 it should compute the difference in seconds from sending until it got confirmed? Or we remove this example altogether

Reply:
Added the duration